### DISCUSSION TITLE
Static Copy not done on Serve

### DISCUSSION THREAD

**Discussion by blueridgemountains1** [sapphi-red/vite-plugin-static-copy#85] [Q&A] [answered]
```
{
    plugins: [
        viteStaticCopy({
            targets: [
                {
                    src: resolve(__dirname, 'node_modules/leaflet/dist/images/*'),
                    dest: resolve(__dirname, 'public/vendor/leaflet/images'),
                    overwrite: true,
                },
            ],
        })
    ],
}
```

Files are copied over just fine during the build process. However, during "serve" I am flashed the message
`[vite-plugin-static-copy] Collected 5 items.`

_However_ , the files don't actually get copied. I have tried to look through the source code but can't figure out why these files fail to copy. As I said, this works fine with `build`.

Any help? Thank you.

**Comment by sapphi-red** [comment, marked as answer]
This plugin does not copy files during dev. They are served directly from dev server.
https://github.com/sapphi-red/vite-plugin-static-copy#differences-with-rollup-plugin-copy
